ui/components/app/confirm-page-container/confirm-page-container-container.test.js

```diff
@@ -119,25 +119,11 @@ describe('Confirm Page Container Container Test', () => {
 
   describe('Contact/AddressBook name should appear in recipient header', () => {
     it('should not show add to address dialog if recipient is in contact list and should display contact name', () => {
-      const addressBookName = 'New contract';
```

Comment:
This change to the unit test is made because as of #15888, [we stopped using the address book contact name in the container file](https://github.com/MetaMask/metamask-extension/pull/15888/files#diff-4e49efdf837e7d7515c99a797b02d7ff25cffd501b31f5d7e7bdadae8d1d910dL23), and instead relied on the `toNickname` passed in from the parent for the sake of rendering the addressbook contact name.